nav2_costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -79,11 +80,13 @@ InflationLayer::onInitialize()
   declareParameter("inflation_radius", rclcpp::ParameterValue(0.55));
   declareParameter("cost_scaling_factor", rclcpp::ParameterValue(10.0));
   declareParameter("inflate_unknown", rclcpp::ParameterValue(false));
+  declareParameter("inflate_around_unknown", rclcpp::ParameterValue(false));
 
   node_->get_parameter(name_ + "." + "enabled", enabled_);
   node_->get_parameter(name_ + "." + "inflation_radius", inflation_radius_);
   node_->get_parameter(name_ + "." + "cost_scaling_factor", cost_scaling_factor_);
   node_->get_parameter(name_ + "." + "inflate_unknown", inflate_unknown_);
```

Comment:
What's this parameter? I haven't see this before: `inflate_unknown`. Did you test to see what this does / if that actually resolves the issue? 

I did not know myself about this parameter. It looks like we ported this from ROS1 but there's never been documentation about it in ROS1 or ROS2. This may be a hidden feature that actually fixes it.

Reply:
If set to true it allows the inflated area to extend into unknown space too else it gets cut of at unknown areas.

Reply:
got it, so its correct to say that do 2 different things.

Reply:
yeah, exactly